src/components/Pagination/Pagination.test.tsx

```diff
@@ -0,0 +1,115 @@
+    const user = userEvent.setup();
+    render(<Pagination numberOfPages={10} currentPage={2} />);
+    const button1 = screen.getByText('1');
+    const button2 = screen.getByText('2');
+    expect(button1).not.toHaveAttribute('aria-current', 'page');
+    expect(button2).toHaveAttribute('aria-current', 'page');
+    await user.click(screen.getByTestId('prevBtn'));
+    expect(button1).toHaveAttribute('aria-current', 'page');
+    expect(button2).not.toHaveAttribute('aria-current', 'page');
+  });
+
+  it('Renders next set of pages when going from page 5 to 6', async () => {
+    const user = userEvent.setup();
+    render(<Pagination numberOfPages={10} currentPage={5} />);
+    expect(screen.getByText('1')).toBeInTheDocument();
+    expect(screen.getByText('5')).toBeInTheDocument();
+    await user.click(screen.getByTestId('nextBtn'));
+    expect(screen.queryByText('1')).not.toBeInTheDocument();
+    expect(screen.queryByText('5')).not.toBeInTheDocument();
+    expect(screen.getByText('6')).toBeInTheDocument();
+    expect(screen.getByText('10')).toBeInTheDocument();
+  });
+
+  it('Renders previous set of pages when going from page 6 to 5', async () => {
+    const user = userEvent.setup();
+    render(<Pagination numberOfPages={10} currentPage={6} />);
+    expect(screen.getByText('6')).toBeInTheDocument();
+    expect(screen.getByText('10')).toBeInTheDocument();
+    await user.click(screen.getByTestId('prevBtn'));
+    expect(screen.queryByText('6')).not.toBeInTheDocument();
+    expect(screen.queryByText('10')).not.toBeInTheDocument();
+    expect(screen.getByText('1')).toBeInTheDocument();
+    expect(screen.getByText('5')).toBeInTheDocument();
+  });
+
+  it('Sets current page to the page that was clicked', async () => {
+    const user = userEvent.setup();
+    render(<Pagination numberOfPages={10} />);
+    const button3 = screen.getByText('3');
+    expect(button3).not.toHaveAttribute('aria-current', 'page');
+    await user.click(button3);
+    expect(button3).toHaveAttribute('aria-current', 'page');
+  });
+
+  it('Keeps current page synced with provided current page', () => {
+    const { rerender } = render(<Pagination numberOfPages={10} currentPage={3} />);
+    expect(screen.getByText('3')).toHaveAttribute('aria-current', 'page');
+    rerender(<Pagination numberOfPages={10} currentPage={5} />);
+    expect(screen.getByText('5')).toHaveAttribute('aria-current', 'page');
+  });
+
+  it('Triggers onChange callback providing page and isCurrent', async () => {
+    const user = userEvent.setup();
+    const mockFn = jest.fn();
+    render(<Pagination numberOfPages={10} onChange={mockFn} />);
+    await user.click(screen.getByTestId('nextBtn'));
+    expect(mockFn).toHaveBeenCalled();
+  });
+
+  it('Renders pages when passed paginationItem prop', () => {
+    const paginationItem = (page: number, isCurrent: boolean) => (
+      <Link href="docs-demo-link" aria-current={isCurrent}>
```

Comment:
looks like links are getting aria-current="true/false" and buttons are getting "page" or no attr. intentional?